dvc/repo/pkg/install.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import unicode_literals
```

Comment:
This whole file should not be here.

Reply:
Woops not sure how that got into this PR. Will fix

Reply:
Removed